.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,20 @@
+name: Chromatic
+
+on: push
+
+jobs:
+  chromatic-deployment:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
is this needed by chromatic?

Reply:
ya, doc says chromatic needs full git history to keep track of changes

Reply:
makes sense